pytests/tests/test_pretrans.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2020 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+#   Author: Keerthana K <keerthanak@vmware.com>
+
+import pytest
+
+@pytest.fixture(scope='module', autouse=True)
+def setup_test(utils):
+    yield
+    teardown_test(utils)
+
+def teardown_test(utils):
```

Comment:
Please erase all the packages installed by this script as a part of cleanup. Apart from this, LGTM